sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_clients_daily_aggregates_v2/schema.yaml

```diff
@@ -0,0 +1,159 @@
+fields:
```

Comment:
Would be great to replace the column name in the descriptions for more explanatory content.

Reply:
@jaredsnyder @florezita any suggestions for descriptions?

Reply:
Sorry I do not have time to add the descriptions but we do badly need them

Reply:
do we need them _before_ we merge PR into production, or is this a blocker to merge?

Reply:
not that we need to merge TODAY but curious to know the level of blocking this does

Reply:
Typically I would consider this blocking

Reply:
Same, we need the schema with descriptions before merging. And even for the review, to understand what data is there. Perhaps this [Ads glossary](https://mozilla-hub.atlassian.net/wiki/spaces/MA1/pages/328565119/Glossary) or @chelseybeck as part of [collecting business definitions](https://mozilla-hub.atlassian.net/browse/AD-807) could help?

Reply:
I have updated the schema descriptions that in my opinion explains the fields and implementations. But its important the descriptions are meaningful to others as well.
@lucia-vargas-a @jaredsnyder  Therefore, please review and help me make the descriptions better if possible. Thank you!

I looked at the [Ads glossary](https://mozilla-hub.atlassian.net/wiki/spaces/MA1/pages/328565119/Glossary) that @lucia-vargas-a suggested. It was last updated in Jan 2024 and seems a bit outdated.

Not sure if HNT has a similar glossary. Some of the names hnt use keep changing (example: item/content/article/stories/...)